YahooTickerDownloader.py

```diff
@@ -130,9 +132,12 @@ def main():
                 data.append(symbol.getRow())
             elif (symbol.exchange == args.Exchange):
                 data.append(symbol.getRow())
-
+                
         with open(downloader.type + '.csv', 'wb') as f:
-            f.write(data.csv.encode('UTF-8'))
+            if sys.version_info.major >= 3:
+                f.write(data.csv.decode('UTF-8'))
```

Comment:
In fact this is now worse as it now never works in python3. Shame on me for not running the code once.